WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -249,8 +253,8 @@ class GutenbergViewController: UIViewController, PostEditor {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        setupGutenbergView()
         createRevisionOfPost()
+        setupGutenbergView()
```

Comment:
This change in line order here, where `setupGutenbergView` is now below  `createRevisionOfPost` is important.

I don't know if this was done for a reason - but changing the order here allowed the autosave content to be loaded into the newly created post revision before the content is loaded into the editor.

Reply:
Looks good to me, great catch!
I don't recall the previous ordering being important for the block editor lifecycle 👍